Projects/App/Sources/Screens/ChatScreen/Models/Message.swift

```diff
@@ -249,6 +251,12 @@ func hash(into hasher: inout Hasher) { hasher.combine(globalId) }
                 type = .gif(url: URL(string: text.text))
             } else if text.text.isCrossSell {
                 type = .crossSell(url: URL(string: text.text))
+            } else if text.text.isDeepLink {
+                if let url = URL(string: text.text) {
```

Comment:
make sure (since we don't support other deep links handling) to set type to deepLink only for contract deep link.
In other cases is still should be text.